tests/components/test_graphite.py

```diff
@@ -19,21 +19,18 @@ class TestGraphite(unittest.TestCase):
     def setup_method(self, method):
         """Setup things to be run when tests are started."""
         self.hass = get_test_home_assistant()
-        self.hass.config.latitude = 32.87336
-        self.hass.config.longitude = 117.22743
         self.gf = graphite.GraphiteFeeder(self.hass, 'foo', 123, 'ha')
 
     def teardown_method(self, method):
         """Stop everything that was started."""
         self.hass.stop()
 
-    @mock.patch('homeassistant.components.graphite.GraphiteFeeder')
-    def test_minimal_config(self, mock_gf):
```

Comment:
I consider it a red flag if tests are being removed, means we're breaking something.